sql/core/src/test/scala/org/apache/spark/sql/types/DataTypeTest.scala

```diff
@@ -0,0 +1,68 @@
+/*
```

Comment:
Let's move the tests to `DataTypeSuite.scala`

Reply:
moved!